src/system/applications/item/base.ts

```diff
@@ -116,6 +116,17 @@ export class BaseItemSheet extends TabsApplicationMixin(
             )
                 formData.set('system.activation.skill', null);
 
+            if (
+                'system.activation.attribute' in formData.object &&
+                formData.object['system.activation.attribute'] === 'default'
+            ) {
+                formData.set(
+                    'system.activation.attribute',
+                    CONFIG.COSMERE.skills[
+                        formData.object['system.activation.skill'] as Skill
```

Comment:
Note: This wasn't possible when the skills were defined as enums

Reply:
This works just fine with the skills defined as an enum? Whether the skills are an enum or not shouldn't have any impact here, as you're just casting the type; the generated js will be identical.

I pulled in the branch and converted skills back to an enum to test and the build works without any further changes.

Reply:
Interesting... I was getting error highlights in the IDE, and confirmed my understanding with a Google that you can't index into an enum with a string and you can't convert the enum key into string either.

I'm away from home for a few days, but will have to try reverting this when I get back.

Reply:
After double checking, I found that just casting the form output to an enum is doable... (odd how it gave me errors before that matched my understanding and reading on the topic 🤔).
So I've reverted the skills for consistency